crates/sui-indexer/src/store/pg_indexer_store.rs

```diff
@@ -852,10 +852,14 @@ impl IndexerStore for PgIndexerStore {
                                 objects::checkpoint.eq(excluded(objects::checkpoint)),
                                 objects::version.eq(excluded(objects::version)),
                                 objects::object_digest.eq(excluded(objects::object_digest)),
+                                objects::owner_type.eq(excluded(objects::owner_type)),
                                 objects::owner_address.eq(excluded(objects::owner_address)),
-                                objects::previous_transaction
-                                    .eq(excluded(objects::previous_transaction)),
+                                objects::previous_transaction.eq(excluded(objects::previous_transaction)),
+                                objects::object_type.eq(excluded(objects::object_type)),
                                 objects::object_status.eq(excluded(objects::object_status)),
+                                objects::has_public_transfer.eq(excluded(objects::has_public_transfer)),
+                                objects::storage_rebate.eq(excluded(objects::storage_rebate)),
+                                objects::bcs.eq(excluded(objects::bcs)),
```

Comment:
would be good to just `set` all other columns, but did not see an easy way of doing this after a quick search.

Reply:
this is so error prone :( , I wish there are macro for diesel to handle this...... maybe we can write one when have time

Reply:
100%, at least some tests to catch missing column updates if that happens.